### PR TITLE
Shift management and handover 

### PR DESCRIPTION
**Pull request by evisochek**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28569

## Description
Shift management and shift handover are crucial processes of SOC team management.
This pack's purpose is Provide a single interface for all those essential elements in one place in order to prevent loss of data in the shift handover process. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [x ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@David-BMS @evisochek Doc review completed.

**Comment by ShirleyDenkberg** [comment]
@Itay4 @glicht @teizenman Doc review completed.

File: Packs/ShiftManagement/IncidentTypes/incidenttype-Shift_handover.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"disabled": false,
+	"fromServerVersion": "",
+	"hours": 0,
+	"hoursR": 0,
+	"id": "Shift handover",
+	"itemVersion": "",
+	"layout": "c6610ca1-32e1-49b2-84c9-239d7e3b405d",
+	"locked": false,
+	"name": "Shift handover",
+	"packID": "27061f91-5a0b-4156-8aeb-c6c28ef1570a",
```
Comment (David-BMS) [inline Packs/ShiftManagement/IncidentTypes/incidenttype-Shift_handover.json:17]: Is this relevant?

File: Packs/ShiftManagement/Layouts/layoutscontainer-shift_handover.json [hunk 2]
```diff
@@ -0,0 +1,758 @@
...
+                "name": "Basic Information",
+                "query": null,
+                "queryType": "",
+                "readOnly": false,
+                "type": ""
+            }
+        ]
+    },
+    "group": "incident",
+    "id": "c6610ca1-32e1-49b2-84c9-239d7e3b405d",
```
Comment (David-BMS) [inline Packs/ShiftManagement/Layouts/layoutscontainer-shift_handover.json:477]: Change ID

File: Packs/ShiftManagement/Layouts/layoutscontainer-shift_handover.json [hunk 3]
```diff
@@ -0,0 +1,758 @@
...
+                "isVisible": true,
+                "name": "Timeline Information",
+                "query": null,
+                "queryType": "",
+                "readOnly": false,
+                "type": ""
+            },
+            {
+                "description": "",
+                "fields": [
```
Comment (David-BMS) [inline Packs/ShiftManagement/Layouts/layoutscontainer-shift_handover.json:561]: Clean non relevant fields.

File: Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml [hunk 4]
```diff
@@ -0,0 +1,392 @@
+id: Assign Active Incidents to Next Shift_V2
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml:1]: Remove _ from name

File: Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Assign Active Incidents to Next Shift_V2
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml:5]: Same

File: Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+description: "This playbook reassigns Active Incidents to the current users on call.\
+  \ It requires shift management to be set up.  The playbook can be run as a job a few minutes after\
+  \ the scheduled shift change time.\n\nYou can update the playbook input with a different\
+  \ search query, if required.  Will branch if there are no incidents that match\
+  \ the query and no users on call. \n\nSearch results are the default 100 incidents\
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml:10]: Is the 100 incident still valid?

File: Packs/ShiftManagement/Playbooks/Set_a_Shift_handover_meeting.yml [hunk 7]
```diff
@@ -0,0 +1,505 @@
+id: Set a Shift handover meeting
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Set_a_Shift_handover_meeting.yml:1]: Name is Set? consider changing to Set Up

File: Packs/ShiftManagement/Playbooks/Set_a_Shift_handover_meeting.yml [hunk 8]
```diff
@@ -0,0 +1,505 @@
...
+version: -1
+name: Set a Shift handover meeting
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Set_a_Shift_handover_meeting.yml:3]: Same

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 9]
```diff
@@ -0,0 +1,1663 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "11":
+    id: "11"
+    taskid: 23e39110-27fc-462f-8abd-c42c0df89a59
+    type: playbook
+    task:
+      id: 23e39110-27fc-462f-8abd-c42c0df89a59
+      version: -1
+      name: Assign Active Incidents to Next Shift_V2
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:198]: Remove _

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 10]
```diff
@@ -0,0 +1,1663 @@
...
+      description: "This playbook reassigns Active Incidents to the current users\
+        \ on call. It requires shift management to be setup. This playbook can be run as a job a\
+        \ few minutes after the scheduled shift change time.\n\nYou can update the playbook\
+        \ input with a different search query if required.  The flow will branch if there\
+        \ are no incidents that match the query or no users on call. \n\nSearch\
+        \ results are the default 100 incidents returned by the query.\n\nCases will\
+        \ not be assigned to users who are defined as OOO (by the OutOfOffice automation)."
+      playbookName: Assign Active Incidents to Next Shift_V2
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:206]: Remove _

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 11]
```diff
@@ -0,0 +1,1663 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "16":
+    id: "16"
+    taskid: 5eaa35ff-ffca-482f-8b58-c0f25ecfedf7
+    type: playbook
+    task:
+      id: 5eaa35ff-ffca-482f-8b58-c0f25ecfedf7
+      version: -1
+      name: Set a Shift handover meeting
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:279]: If name changes update

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 12]
```diff
@@ -0,0 +1,1663 @@
...
+      playbookName: Set a Shift handover meeting
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:280]: Same

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 13]
```diff
@@ -0,0 +1,1663 @@
...
+    simple: "Yes"
+  required: false
+  description: "Yes - If you want the playbook to reassign active incidents\
+    \ to on-call users. The playbook will take into consideration users that are out\
+    \ of the office.\nNo- If you do not want to reassign active incidents to on-call users. The current incident\
+    \ owner will remain. \n"
+  playbookInputQuery:
+- key: IncidentSearchQuery
+  value:
+    simple: 'status:active  -category:job '
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:1620]: Leave out the incident type for shift handover

File: Packs/ShiftManagement/Playbooks/Shift_handover_README.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+- A SOC manager briefing
+- A display of the active incidents, team members that are on-call, and team members that are out of the office.
+By modifying the playbook inputs, you can decide whether to activate the Assign Active Incidents to Next Shift and whether a user who is out of the office will take in consideration. 
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Set a Shift handover meeting
+* Assign Active Incidents to Next Shift_V2
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover_README.md:13]: Remove _

File: Packs/ShiftManagement/README.md [hunk 15]
```diff
@@ -0,0 +1,10 @@
+Shift management and shift handover are crucial processes of SOC team management.
+This pack's purpose is Provide a single interface for all those essential elements in one place in order to prevent loss of data in the shift handover process.
```
Comment (David-BMS) [inline Packs/ShiftManagement/README.md:2]: Upper case P in provided not needed.

File: Packs/ShiftManagement/ReleaseNotes/1_1_5.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
+
+#### Incident Fields
+- **Out off the office**
+- **Shift open incidents**
+- **To join the meeting**
+- **To start the meeting**
+- **Shift manager briefing **
```
Comment (David-BMS) [inline Packs/ShiftManagement/ReleaseNotes/1_1_5.md:7]: Fix markdown

File: Packs/ShiftManagement/ReleaseNotes/1_1_5.md [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+
+#### Playbooks
+##### New: Set a Shift handover meeting
+- This playbook is used to create an online meeting for shift handover. Currently, this playbook supports Zoom. (Available from Cortex XSOAR 6.0.0).
+##### New: Assign Active Incidents to Next Shift_V2
+- This playbook reassigns Active Incidents to the current users on call. It requires shift management to be set up.  The playbook can be run as a job a few minutes after the scheduled shift change time.
+
+You can update the playbook input with a different search query, if required.  Will branch if there are no incidents that match the query and no users on call. 
+
+Search results are the default 100 incidents returned by the query.
```
Comment (David-BMS) [inline Packs/ShiftManagement/ReleaseNotes/1_1_5.md:20]: Is this still relevant?

File: Packs/ShiftManagement/pack_metadata.json [hunk 18]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Shift Management",
-    "description": "Shift management related content",
+    "description": "This pack's purpose is to provide a single interface for all those essential elements of Shift management and handover in one place.",
     "support": "xsoar",
-    "currentVersion": "1.1.3",
+    "currentVersion": "1.1.5",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
```
Comment (David-BMS) [inline Packs/ShiftManagement/pack_metadata.json:8]: Add
Dependencies
Keywords
Tags
Categories

**Review by David-BMS — CHANGES_REQUESTED** [review]

File: Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml [hunk 19]
```diff
@@ -0,0 +1,392 @@
...
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: c9aebe9c-0692-4f6c-8454-157709aa68b8
+    type: regular
+    task:
+      id: c9aebe9c-0692-4f6c-8454-157709aa68b8
+      version: -1
+      name: Get Users on Call
+      description: commands.local.cmd.get.users
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml:177]: @ShirleyDenkberg this is a server default description and not the correct description.

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 20]
```diff
@@ -0,0 +1,1663 @@
...
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: 9c8c9a00-b552-4178-8ea7-2fdf55284b23
+    type: regular
+    task:
+      id: 9c8c9a00-b552-4178-8ea7-2fdf55284b23
+      version: -1
+      name: Close previous change shift incident
+      description: commands.local.cmd.close.inv
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:157]: @ShirleyDenkberg this is a server default description and not the correct description.

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 21]
```diff
@@ -0,0 +1,1663 @@
...
+    quietmode: 0
+  "30":
+    id: "30"
+    taskid: 26c61842-7348-4d51-8c9f-bfa1915a552f
+    type: regular
+    task:
+      id: 26c61842-7348-4d51-8c9f-bfa1915a552f
+      version: -1
+      name: 'Set apps Channel, team name and message to incident field '
+      description: commands.local.cmd.set.incident
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:660]: @ShirleyDenkberg this is a server default description and not the correct description.

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 22]
```diff
@@ -0,0 +1,1663 @@
...
+    quietmode: 0
+  "34":
+    id: "34"
+    taskid: 4c8871e0-9c90-4758-846b-d09c156c67f2
+    type: regular
+    task:
+      id: 4c8871e0-9c90-4758-846b-d09c156c67f2
+      version: -1
+      name: Set SOC manager briefing for the team
+      description: commands.local.cmd.set.incident
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:1187]: @ShirleyDenkberg this is a server default description and not the correct description.

File: Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml [hunk 23]
```diff
@@ -0,0 +1,392 @@
...
+        "width": 850,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: IncidentSearchQuery
+  value:
+    simple: 'status:active  -category:job '
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml:384]: make sure to exclude the shift handover type

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 24]
```diff
@@ -0,0 +1,1669 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "16":
+    id: "16"
+    taskid: 5eaa35ff-ffca-482f-8b58-c0f25ecfedf7
+    type: playbook
+    task:
+      id: 5eaa35ff-ffca-482f-8b58-c0f25ecfedf7
+      version: -1
+      name: Set up a Shift handover meeting
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:281]: Validate skip works

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 25]
```diff
@@ -0,0 +1,1669 @@
...
+    quietmode: 0
+  "11":
+    id: "11"
+    taskid: 23e39110-27fc-462f-8abd-c42c0df89a59
+    type: playbook
+    task:
+      id: 23e39110-27fc-462f-8abd-c42c0df89a59
+      version: -1
+      name: Assign Active Incidents to Next Shift V2
+      description: "This playbook reassigns Active Incidents to the current users\
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:200]: There should be a link to the Job configuration article.

File: Packs/ShiftManagement/Playbooks/Shift_handover.yml [hunk 26]
```diff
@@ -0,0 +1,1669 @@
...
+- key: AssignActiveIncidentsToNextShift
+  value:
+    simple: "Yes"
+  required: false
+  description: "Yes - If you want the playbook to reassign active incidents to on-call\
+    \ users. The playbook will take into consideration users that are out of the office.\n\
+    No- If you do not want to reassign active incidents to on-call users. The current\
+    \ incident owner will remain. \n"
+  playbookInputQuery:
+- key: IncidentSearchQuery
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover.yml:1620]: If input is not needed remove it

File: Packs/ShiftManagement/Playbooks/Shift_handover_README.md [hunk 27]
```diff
@@ -0,0 +1,51 @@
...
+* setIncident
+* closeInvestigation
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| AssignActiveIncidentsToNextShift | Yes - If you want the playbook will reassign active incidents to on-call users. The playbook will take into consideration users who are out of the office.<br/>No- If you do not want to reassign active incidents to on-call users. The current incident owner will remain. <br/> | Yes | Optional |
+| IncidentSearchQuery | Query for the incidents to reassign.  Incidents need to be active. The playbook will not reassign pending incidents.  | status:active  -category:job  | Optional |
```
Comment (David-BMS) [inline Packs/ShiftManagement/Playbooks/Shift_handover_README.md:36]: Update accordingly.

**Review by David-BMS — APPROVED** [review]
We saw an issue with the OOO widget please review

Need to add playbook images to readmes

Make sure to test skip branch for meeting subplaybook

**Comment by evisochek** [comment]
We saw an issue with the OOO widget please review - Fixed by @adi88d 

Need to add playbook images to readmes - Done.

Make sure to test skip branch for meeting subplaybook - Done.

**Pull request merged**